Makefile

```diff
@@ -120,11 +120,12 @@ endif
 
 GO_SOURCES_OWN := $(filter-out vendor/% %/bindata.go, $(GO_SOURCES))
 
-#To update swagger use: GO111MODULE=on go get -u github.com/go-swagger/go-swagger/cmd/swagger@v0.20.1
+#To update swagger use: GO111MODULE=on go get -u github.com/go-swagger/go-swagger/cmd/swagger
```

Comment:
Why?

Reply:
just noice - wothout the tag it is more confinient to go directly automaticaly to the latest stable

I also thought about removing it but it will not hurt anyone where it is so here it stays